static/src/stylesheets/module/experiments/_acquisitions-epic-interactive.scss

```diff
@@ -0,0 +1,21 @@
+.contributions__epic.contributions__epic--interactive {
+    padding: ($gs-baseline / 3) 0 $gs-baseline;
+}
+
+.contributions__epic-interactive-wrapper {
+    padding: 0 20px;
+    max-width: 620px;
+    clear: both
+}
+
+@media (min-width: 71.25em) {
```

Comment:
Is there a chance we can use the `sass-mq` mixin instead ([example](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/layout/_ab-new-header-test-variant.scss#L24))?